docs/user_guide/intelligence_dashboard.md

```diff
@@ -8,11 +8,16 @@ parent: User Guide
 
 # Intelligence 
 
-Cyences version 1.2.0 introduces a new drop-down option named Intelligence in the app's navigation bar and it contains the following items:
+* Cyences version 1.2.0 introduces a new drop-down option named Intelligence in the app's navigation bar and it contains the following items:
 
-* Asset Intelligence
-* Device Inventory
-* Globally Detected Malicious IPs
+    * Asset Intelligence
```

Comment:
List all new version:
-> device inventory
-> user inventory
-> intelligence
globally detected malicious ips

Reply:
It's version wise high level details. I've added intelligence and user inventory in second point.
![image](https://github.com/CrossRealms/Splunk-Cyences-App-for-Splunk/assets/57986609/5b022613-60a1-4066-be52-7c488100f46f)

Reply:
Done